main/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class UserProfile(models.Model):
```

Comment:
Why you create class with only OneToOneField?

Reply:
To further creation of UserProfile. Currently it isn't used in fact